tests/server/mqttd.c

```diff
@@ -637,6 +637,8 @@ static curl_socket_t mqttit(curl_socket_t fd)
 
   end:
   fclose(dump);
```

Comment:
```suggestion
  if(dump)
    fclose(dump);
```

Reply:
Yeah, I didn't touch line that since it isn't relevant for this fix but you're correct and I'll fix while at it!